vault_install.sh

```diff
@@ -28,7 +36,8 @@ then
   echo -e "\n\033[1;32m==== Hashicorp repo present ====\033[0m\n"
 else
   echo -e "\n\033[1;33m==== Adding Hashicorp repo ====\033[0m\n"
-  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] \
+  https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
   sudo apt update
```

Comment:
If you have the apt update above, why are you doing this here? Do you know about the apt-cache? Could you just check if the package is in the apt-cache and update if not?

Reply:
This is from the Linux vault install docs. Maybe I’m misunderstanding how this works.